ui/hooks/useAccountTotalCrossChainFiatBalance.ts

```diff
@@ -39,7 +39,11 @@ export const useAccountTotalCrossChainFiatBalance = (
 ) => {
   const allNetworks = useSelector(getNetworkConfigurationsByChainId);
 
-  const enabledNetworksByNamespace = useSelector(getEnabledNetworksByNamespace);
+  const selectedEnabledNetworks = useSelector(getEnabledNetworks);
+  const enabledNetworksByNamespace = useMemo(
+    () => Object.assign({}, ...Object.values(selectedEnabledNetworks)),
+    [selectedEnabledNetworks],
+  );
```

Comment:
Before - we were only getting the enabled chains for the selected network type. So if you select solana, it will not contain the EIP155 chain information.

After - now we fetch the entire object for selected chains across network types (EIP155, and solana). We flatten this and therefore can be used to get the correct balance for EVM accounts in the list.